torchgeo/datasets/levircd.py

```diff
@@ -0,0 +1,190 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""LEVIR-CD+ dataset."""
+
+import glob
+import os
+from typing import Callable, Dict, Optional
+
+import numpy as np
+import torch
+from PIL import Image
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import download_and_extract_archive
+
+
+class LEVIRCDPlus(VisionDataset):
+    """LEVIR-CD+ dataset.
+    The `LEVIR-CD+ <https://github.com/S2Looking/Dataset>`_
+    dataset is a dataset for building change detection.
+
+    Dataset features:
+    * image pairs of 20 different urban regions across Texas between 2002-2020
+    * binary change masks representing building change
+    * three spectral bands - RGB
+    * 985 image pairs with 50 cm per pixel resolution (~1024x1024 px)
+
+    Dataset format:
+    * images are three-channel pngs
+    * masks are single-channel pngs where no change = 0, change = 255
+
+    Dataset classes:
+    1. no change
+    2. change
+
+    If you use this dataset in your research, please cite the following paper:
+    * https://arxiv.org/abs/2107.09244
+    """  # noqa: E501
+
+    url = "https://drive.google.com/file/d/1JamSsxiytXdzAIk6VDVWfc-OsX-81U81/view?usp=sharing"
+    md5 = "1adf156f628aa32fb2e8fe6cada16c04"
+    filename = "LEVIR-CD+.zip"
+    filenames = ["train", "test"]
```

Comment:
I would call this `splits` to help disambiguate from `filename`.

Reply:
Renamed to `splits`